recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - typer
     - authlib
     - psycopg2
-    - httpx >=0.18.2
+    - httpx >=0.20.0
```

Comment:
i'm unsure about this pinning. i'd suggest to be a bit more loose (instead of `==0.20.0`), hoping that future versions won't break quetz. also httpx seems to be on its way to 1.0.0 (https://github.com/encode/httpx/releases/tag/1.0.0.beta0), so hopefully soon we can just pin the major version.

Reply:
yeah we had issues in the past with `authlib` and `httpx` IIRC, which is why we started to be a bit more strict about those. I think we can loosen them now.